### DISCUSSION TITLE
Combine two RWMol Objects

### DISCUSSION THREAD

**Discussion by JanoschMenke** [rdkit/rdkit#6651] [Q&A] [answered]
Hi Guys,

is there a straight forward way to add to one RWMol Object another molecule (a RWMol object as well)
Like there is an `addAtom()` functionality.

**Comment by ptosco** [comment, marked as answer]
Have a look at [CombineMols](https://www.rdkit.org/docs/source/rdkit.Chem.rdmolops.html#rdkit.Chem.rdmolops.CombineMols)

**Reply by JanoschMenke** [reply]
Thanks that works easier than i thought
